docs/agents/sematext-agent/custom-logs.md

```diff
@@ -0,0 +1,179 @@
+title: Sematext Agent Custom Logs Configuration
+description: Custom Logs Configuration for Sematext Agent
+
+Sematext Agent supports configuring log shipping through a YAML file, `custom-logs.yml`. This feature allows users to specify log file locations and custom parsing rules, which is useful for log files not [discovered automatically](https://sematext.com/docs/fleet/discovery/) or when custom log parsing rules are needed.
+
+## How it works
+
+The Sematext Agent periodically checks the `custom-logs.yml` file. If valid log shipping rules are found, the agent starts collecting, parsing, and shipping logs from the specified files to the Sematext Cloud Logs App that matches the provided App token. To stop log shipping, remove the rule from the file.
+
+## Requirements
+
+Sematext Agent [version 3.6.0](https://sematext.com/docs/agents/sematext-agent/releasenotes/#version-360) and newer.
+
+## How to start
+
+[Create a new Logs App](https://sematext.com/docs/logs/quick-start/) in Sematext Cloud. If you have custom log files, create a [Generic Logs App](https://sematext.com/docs/integration/generic-logs-integration/). For log files generated from one of our [supported integrations](https://sematext.com/docs/integration/), create a Logs App of the relevant type.
+
+As soon as the Sematext Agent is installed, ignore the Discovery screen and keep a copy of the App token. To retrieve this token, navigate to the **Ship Logs** screen from the left Menu panel and select **Custom Integrations**.
+
+The next step is to create the `custom-logs.yml` file and place it in the appropriate directory (see below) in the agent configuration.
```

Comment:
STA will probably come with a sample custom-logs.yml, but with everything commented out. If so, then create --> edit.

Reply:
Yes they will have a sample file.
TODO(bora): 
1. 
``` 
  cp ./configs/st-agent.yml $IMAGE_PATH/opt/spm/properties/st-agent.yml
  cp ./configs/autodisco.yml $IMAGE_PATH/opt/spm/properties/autodisco.yml
...
  getGithubResource "st-agent.yml" "$IMAGE_PATH/opt/spm/properties/st-agent.yml"
  getGithubResource "autodisco.yml" "$IMAGE_PATH/opt/spm/properties/autodisco.yml"
```
2.
```
cp $SPM_HOME/properties/st-agent.yml $DIAG_DIR/st-agent 2> /dev/null
cp $SPM_HOME/properties/autodisco.yml $DIAG_DIR/st-agent 2> /dev/null
```

Reply:
TODO(bora): spm files and diag

Reply:
Updated. Keeping this unresolved until Bora is ready.

Reply:
https://github.com/sematext/spm-client/pull/147